arch/risc-v/src/esp32c3/Kconfig

```diff
@@ -261,6 +261,10 @@ config ESP32C3_SPIFLASH
 	select MTD_BYTE_WRITE
 	select MTD_PARTITION
 
+config ESP32C3_SPIFLASH_BLOCK_SIZE
+	int "SPI FLASH BLOCK SIZE"
+	default 64
```

Comment:
Should the default be 256?

Reply:
no, because the littlefs chche size default value is 4 * flash_block_size, prog size default value is flash_block_size, prog size smaller, flash memory usage higher.

Reply:
The flash block size is not configurable, so it is wrong to create a configuration for it.
The `MTD_BLKSIZE` macro retrieves this info from the `page_size` field which is configured by the bootloader depending on the underlying SPI Flash IC present on the module.

It seems that, instead of doing this change on the driver, the correct approach would be to configure the scale factor on LittleFS accordingly.